ui/src/app/streams/stream-deploy/stream-deploy.component.ts

```diff
@@ -16,6 +16,7 @@ import {saveAs} from 'file-saver/FileSaver';
 import {BsModalService} from 'ngx-bootstrap';
 import {AppPropertiesSource, StreamDeployAppPropertiesComponent} from './app-properties/app-properties.component';
 import {Properties} from 'spring-flo';
+import moment = require('moment');
```

Comment:
I would probably prefer importing moment via `import * as moment from 'moment';`. That looks a bit more TypeScripty.